library/src/main/java/com/phillipcalvin/iconbutton/IconButton.java

```diff
@@ -65,34 +66,47 @@ protected void onLayout(boolean changed, int left, int top, int right, int botto
         textPaint.getTextBounds(text, 0, text.length(), bounds);
 
         int textWidth = bounds.width();
-        int contentWidth = drawableWidth + iconPadding + textWidth;
-
+        int factor = (drawablePosition == DrawablePositions.BOTH) ? 2 : 1;
+        int contentWidth = drawableWidth + iconPadding * factor + textWidth;
         int contentLeft = (int) ((getWidth() / 2.0) - (contentWidth / 2.0));
```

Comment:
This should probably be renamed `sidePadding` or `horizontalPadding` since it's used on both the left and right now.